### PR TITLE
Added support for the Montserrat Font as discussed in #3201

### PR DESCRIPTION
**Pull request by 00SteinsGate00**
As discussed in #3201 I added the Montserrat Font to the list of fonts. It's clean and elegant.

I chose a pretty thing version (namely **Montserrat Light**) as the base version and **Montserrat Regular** for the *bold* version because I think the thinner font looks better. I hope that's fine.

bildschirmfoto 2017-06-21 um 22 00 13

It seems that my Atom Editor removed a lot of trailing whitespaces, this seems to be the default option of Atom and it's supposed to be that way as discussed [here](https://github.com/atom/whitespace/issues/10)

I hope this is not a problem.

### CODE REVIEW

**Comment by lpagliari** [comment]
@00SteinsGate00 apparently when I've accepted #3206 this one started having code conflicts -- probably due to the trailing whitespaces you've removed -- no problem about that, BTW, I do the same thing here.

Could you fix the conflicts so I can merge your changes, please? Thanks, and sorry for taking so long to take a look on your PRs.

**Comment by 00SteinsGate00** [comment]
Sure will do :)
Might have conflicts because they were both branched from the same commit and I didn't base this one on the other.

Don't worry, I can imagine it's a lot of work to maintain a project this big.

**Comment by 00SteinsGate00** [comment]
Done :)

File: src/static/js/pad.js [hunk 1]
```diff
@@ -554,10 +554,10 @@ var pad = {
         pad.changeViewOption('rtlIsTrue', true);
       }
 
-      var fonts = ['useMonospaceFont', 'useOpenDyslexicFont', 'useComicSansFont', 'useCourierNewFont', 'useGeorgiaFont', 'useImpactFont',
+
+      var fonts = ['useMonospaceFont', 'useMontserratFont', 'useOpenDyslexicFont', 'useComicSansFont', 'useCourierNewFont', 'useGeorgiaFont', 'useImpactFont',
         'useLucidaFont', 'useLucidaSansFont', 'usePalatinoFont', 'useRobotoMonoFont', 'useTahomaFont', 'useTimesNewRomanFont',
-        'useTrebuchetFont', 'useVerdanaFont', 'useSymbolFont', 'useWebdingsFont', 'useWingDingsFont', 'useSansSerifFont',
-        'useSerifFont'];
```
Comment (lpagliari) [inline src/static/js/pad.js]: Hum, were these two lines removed on your rebase? They shouldn't...

File: src/static/js/pad_editor.js [hunk 2]
```diff
@@ -30,10 +30,10 @@ var padeditor = (function()
   var settings = undefined;
 
   // Array of available fonts
-  var fonts = ['useMonospaceFont', 'useOpenDyslexicFont', 'useComicSansFont', 'useCourierNewFont', 'useGeorgiaFont', 'useImpactFont',
+
+  var fonts = ['useMonospaceFont', 'useMontserratFont', 'useOpenDyslexicFont', 'useComicSansFont', 'useCourierNewFont', 'useGeorgiaFont', 'useImpactFont',
     'useLucidaFont', 'useLucidaSansFont', 'usePalatinoFont', 'useRobotoMonoFont', 'useTahomaFont', 'useTimesNewRomanFont',
-    'useTrebuchetFont', 'useVerdanaFont', 'useSymbolFont', 'useWebdingsFont', 'useWingDingsFont', 'useSansSerifFont',
-    'useSerifFont'];
```
Comment (lpagliari) [inline src/static/js/pad_editor.js]: Same here

File: src/static/js/pad.js (hunk 1 above)
Comment (00SteinsGate00) [inline src/static/js/pad.js]: Oh it seems like it, sorry. I was merging the lines were the two fonts were added together and I guess those 2 lines kinda slipped. Very sorry about that.

Gonna add them again (hope there's some --amend functionality here on github ), as soon as I can, I'm in class right now and don't have my laptop with me.

Comment (lpagliari) [inline src/static/js/pad.js]: No problem at all. I can squash all the commits when I merge the PR

**Comment by 00SteinsGate00** [comment]
Okay all the fonts are back in order.

**Comment by lpagliari** [comment]
Thanks @00SteinsGate00 !

**Pull request merged**